### PR TITLE
Separate Settings for Different Environment

### PR DESCRIPTION
**Pull request by BolajiOlajide**
#### What Does This PR Do?
Create separate settings for `development` and `production`.

#### Any Background Context You Want To Provide?
The application will be running on different environments:
- Development
This is mainly for development and some settings are specific to development and can't be available on production.
- Production
This is for HEROKU.

#### How can this be manually tested?
By starting the application.

### CODE REVIEW

File: reactlibapp/reactlibapp/settings/production.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# Parse database configuration from $DATABASE_URL
+import dj_database_url
+
+DEBUG = False
+
+DATABASES = {
+    'default': dj_database_url.config()
+}
+
+ALLOWED_HOSTS = ['*']
```
Comment (andela-sjames) [inline reactlibapp/reactlibapp/settings/production.py]: This config looks incomplete though, I can't see any `procfile` yet, also did you check production settings by testing with `gunicorn` locally as it is a replica of Heroku runtime environment. 

https://devcenter.heroku.com/articles/django-app-configuration
http://docs.gunicorn.org/en/stable/install.html

We could either do that now or later when we are ready to push to heroku. 

Aside that, this looks good to me.

Comment (BolajiOlajide) [inline reactlibapp/reactlibapp/settings/production.py]: I agree with you @andela-sjames. I'm trying to start the application with `gunicorn` with the command `gunicorn reactlibapp.manage:app`.
It keeps returning an error saying it failed to find the application `reactlibapp`. Can we pair on this later?

Comment (BolajiOlajide) [inline reactlibapp/reactlibapp/settings/production.py]: cc @andela-sjames

Comment (andela-sjames) [inline reactlibapp/reactlibapp/settings/production.py]: When will you be free for us to look into this @BolajiOlajide

Comment (BolajiOlajide) [inline reactlibapp/reactlibapp/settings/production.py]: In the next one hour, or tomorrow. 
I'll buzz you.

**Comment by BolajiOlajide** [comment]
Hey @andela-sjames @andela-pessien 
I think this looks LGTM.

**Pull request merged**